packages/app/src/app/pages/Dashboard/Content/routes/Templates/Navigation/Navigation.tsx

```diff
@@ -15,6 +15,7 @@ export const Navigation = ({ teamId, number }: INavigationProps) => (
     >
       {teamId ? 'Team Templates' : 'My Templates'}
     </NavigationLink>
-    {number != undefined && <Number>{number}</Number>}
+
+    {number && <Number>{number}</Number>}
```

Comment:
Oh, small thing I found here @MichaelDeBoey. This check will cause `<Number>` not to show when value is `0`, because `0` is falsy.